scripts/performance/perf_load/perf_gen_req_parser.py

```diff
@@ -9,17 +9,21 @@
 from perf_load.perf_req_gen_cfg_writes import RGConfigChangeState
 from perf_load.perf_req_gen_demoted_node import RGPoolNewDemotedNode
 from perf_load.perf_req_gen_get_txn import RGGetTxn
+from perf_load.perf_req_gen_get_taa import RGGetTAA
+from perf_load.perf_req_gen_get_taa_aml import RGGetTAAAML
+from perf_load.perf_req_gen_get_auth_rules import RGGetAuthRules
 from perf_load.perf_req_gen_fees import RGGetFees
 
 
 class ReqTypeParser:
-    _supported_requests =\
+    _supported_requests = \
         {"nym": RGNym, "schema": RGSchema, "attrib": RGAttrib, "cred_def": RGDefinition, "revoc_reg_def": RGDefRevoc,
          "revoc_reg_entry": RGEntryRevoc, "get_nym": RGGetNym, "get_attrib": RGGetAttrib, "get_schema": RGGetSchema,
          "get_cred_def": RGGetDefinition, "get_revoc_reg_def": RGGetDefRevoc, "get_revoc_reg": RGGetEntryRevoc,
          "get_revoc_reg_delta": RGGetRevocRegDelta, "get_payment_sources": RGGetPaymentSources, "payment": RGPayment,
          "verify_payment": RGVerifyPayment, "cfg_writes": RGConfigChangeState, "demoted_node": RGPoolNewDemotedNode,
-         "get_txn": RGGetTxn, "get_fees": RGGetFees}
+         "get_txn": RGGetTxn, 'get_taa': RGGetTAA, 'get_taa_aml': RGGetTAAAML, 'get_auth_rule': RGGetAuthRules,
+         "get_fees": RGGetFees}
```

Comment:
Please extend `scripts/performance/perf_load/test/test_parser.py` with the new requests support.

Reply:
ok